packages/api-explorer/webpack.config.js

```diff
@@ -18,6 +20,11 @@ module.exports = {
       amd: 'react-dom',
       umd: 'react-dom',
     },
+
+    // `@readme/ui` loads this in for the search components but unfortunately we aren't able to exclude this from being
+    // compiled into our dist, despite not using that component. Tf we treat it as a Webpack external it will thankfully
+    // be ignored.
+    'react-instantsearch-dom': 'react-instantsearch-dom',
```

Comment:
This feels a little sketchy, but it seems to work?

Reply:
lol this is nothing as far as ridiculous webpack `externals` go. check out [the maniacal shit](https://github.com/readmeio/readme/blob/54cd7fe31736b82400f879b76b93583a33a7d5b1/packages/react/webpack.config.js#L14-L43) I'm doing in the React subpackage...

Reply:
By which I mean I believe this is generally the correct way to externalize. Can we not do something like this for our syntax highlighter? Or is that verboten cause deps of deps rely on it? Oy vey...

Reply:
@rafegoldberg Yeah both the explorer and `@readme/ui` require `@readme/oas-to-snippet` which uses the syntax highlighter. I'm going to play around with another PR to try to remove it from the dist, but it seemed a bit much on top of everything I'd already done here.

Reply:
@rafegoldberg Since `@readme/ui` is deprecated, thoughts on us forking the tutorial components we're using from it into this repo?

Reply:
Honestly, the thought of forking the tutorial component makes me shudder. The [component architecture](https://github.com/readmeio/readme/tree/next/packages/react/src/routes/Tutorials/components) is quite complex, with a not-insignificant [number of](https://github.com/readmeio/readme/blob/6cdf46723176694ffccc3701826811bfaa1c6276/packages/react/src/core/hooks/useReadmeApi.js) [cross-dependencies](https://github.com/readmeio/readme/blob/0c2509b20e3e18de02246e374c438aee593d9208/packages/react/src/ui/Modal/index.js#L0-L1)... Plus, post-migration (🎫 RM-219), this component _already_ lives in two places!

Considering the [big rewrite](https://linear.app/readme-io/project/20b9da3b992e) we're about to undertake (🎫 RM-246, specifically) I think my line here is "why kill ourselves doing now what we'll just have to redo/refigure a month or two down the line?" Plus I think the colocation of the Explorer and the React app will redound to our benefit in these exact kind of "component sharing" scenarios! In short, I'd hold off on this till we're further in to the refactor.

Reply:
Ended up ripping UI components out of the Explorer completely and are passing them in as props now. #819